test/unit/io.js

```diff
@@ -861,6 +861,54 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression does not throw error', function () {
```

Comment:
Thanks for adding some failure scenarios to the tests. Looking forward to seeing some more positive scenarios/results too.